extensions/amp-next-page/0.2/page.js

```diff
@@ -42,8 +42,17 @@ export class Page {
    * @param {string} url
    * @param {string} title
    * @param {string} image
+   * @param {!PageState=} optInitState
+   * @param {!VisibilityState=} optInitVisibility
    */
-  constructor(manager, url, title, image) {
+  constructor(
+    manager,
+    url,
+    title,
+    image,
+    optInitState = PageState.QUEUED,
```

Comment:
Drop `opt` from the name of these variables. We loosely follow the Google JS style guide which says to not include them: https://google.github.io/styleguide/jsguide.html#features-functions-parameter-return-types. You'll see them elsewhere in the code base as the style guide used to require them in the name.